snet_cli/arguments.py

```diff
@@ -249,6 +280,81 @@ def add_contract_options(parser):
         add_contract_function_options(contract_p, contract_name)
 
 
+def _add_service_publish_arguments(parser):
+    parser.add_argument("--no-register", action="store_true", help="does not register the published service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
```

Comment:
`CONFIG_FILE` metavar doesn't match the documentation

Reply:
Fixed